server/api/users.js

```diff
@@ -62,10 +62,11 @@ router.post("users.info", auth(), async (ctx) => {
 
 router.post("users.update", auth(), async (ctx) => {
   const { user } = ctx.state;
-  const { name, avatarUrl } = ctx.body;
+  const { name, avatarUrl, language } = ctx.body;
 
   if (name) user.name = name;
   if (avatarUrl) user.avatarUrl = avatarUrl;
+  if (language) user.language = language;
```

Comment:
Note: We'll want to validate in the model that the `language` key is one of the available languages, not a blocker for initial landing.

Reply:
Good point, i will connect it with the env variables